test/acceptance/dashboard_test.clj

```diff
@@ -111,3 +122,75 @@
       (t/is (= ["First" "Second" "Third"] titles)))))
 
 
+(defn fs-create-seder
+  "Pre: takes a user, a title, and a path to a Haggadah
+  Post: returns the id of the Seder created"
+  [user title path]
+  (-> (FirestoreClient/getFirestore)
+      (.collection "users")
+      (.document user)
+      (.collection "seders")
+      (.add  (w/stringify-keys (assoc {} :createdAt (java.time.Instant/now)
+                                      :title title
+                                      :haggadah-path path)))
+      (.get)
+      (.getId)))
+
+(defn haggadah-path
+  "Pre: takes a user and a Seder id
+  Post: returns the location of the Haggadah used for the Seder"
+  [user id]
+  (-> (FirestoreClient/getFirestore)
+      (.collection "users")
+      (.document user)
+      (.collection "seders")
+      (.document id)
+      (.get)
+      (.get)
+      (.getString "haggadah-path")))
+
+(defn wait-for-sedarim
+  []
+  (wait-for-collection :sedarim))
+
+(defn create-seder
+  "Pre: takes a title for the Seder
+  Post: returns the id of the Seder created"
+  [title]
+  (doto driver
+    (e/click-visible {:data-testid :create-seder})
+    (e/wait-visible {:id :seder-title})
+    (e/fill  {:id :seder-title} k/home (k/with-shift k/end) k/delete)
+    (e/fill-human {:id :seder-title} title {:mistake-prob 0})
+    (e/click {:data-testid :submit})))
+
+(defn id-and-title
+  "Pre: takes a Seder
+  Post: returns the id and title of the seder in a map"
+  [seder]
+  (let [title (e/get-element-text-el driver seder)
+        id (e/get-element-attr-el driver seder :data-testid)]
+    {:title title :id id}))
+
+(defn all-sedarim
+  "Pre: takes nothing
+  Post: returns the ids and titles of every Seder on the dashboard"
+  []
+  (let [sedarim (e/query-all driver {:fn/has-class :seder-link})]
+    (map id-and-title sedarim)))
+
+(t/deftest create-seder-test
+  (t/testing "When the current user has a Haggadah and creates a Seder which uses that Haggadah, a new Seder with the same Haggadah will appear in the user's collection of Sedarim in firestore"
```

Comment:
NIT: Perhaps the test shouldn't read like it is about _Firestore_ and more describing the concept (not the implementation).

Reply:
I'll change the test description so it describes the general idea of the seder being stored and omits the implementation details.